pages/login.tsx

```diff
@@ -18,6 +20,33 @@ const Login = () => {
     email: "",
     password: "",
   });
+
+  const success = () => {
+    toast.success("Sucessfully Login", {
+      position: "bottom-right",
+      autoClose: 5000,
+      hideProgressBar: false,
+      closeOnClick: true,
+      pauseOnHover: true,
+      draggable: true,
+      progress: undefined,
+      theme: "light",
```

Comment:
you don't need to provide these props every time if you move `ToastContainer` in the `page` where `<App />` is there. And put this `ToastContainer` there. Please read the documentation of `react-toastify` properly and then make the changes

Reply:
Thanks for your suggestions I'll be doing the required changes